accord-core/src/main/java/accord/local/Status.java

```diff
@@ -457,16 +535,24 @@ public enum LogicalEpoch
      */
     public enum Durability
     {
-        NotDurable, Local, Majority, Universal, DurableOrInvalidated;
+        NotDurable, Local, MajorityOrInvalidated, Majority, UniversalOrInvalidated, Universal;
```

Comment:
Right, `OrInvalidated` means we know it reached some decision, we just don't know the actual outcome. So rather than everything being truncated, it means it's been erased, or perhaps never reached quorum and a superseding sync point has been applied.